backend/tests/regression/answer_quality/search_test_config.yaml

```diff
@@ -0,0 +1,11 @@
+
+branch: main
+model_server_ip: "18.218.165.17"
+model_server_port: "80"
+zipped_documents_file: "/Users/danswer/testdocuments/samplepptx.pptx"
+questions_file: "/Users/danswer/danswer/backend/tests/regression/answer_quality/sample_questions.yaml"
```

Comment:
I think local paths is best generally for this stuff, that way everything can be kept in the scope of the Danswer repo?

Under /home (~/) directory is often used as well, no strong preference

but let's not set a file that almost all people will not have like /Users/danswer